src/DrawerLayout.js

```diff
@@ -151,10 +151,12 @@ export default class DrawerLayout extends Component {
           {this.props.children}
         </Animated.View>
 
-        {drawerShown &&
-          <TouchableWithoutFeedback onPress={this._onOverlayClick}>
-            <Animated.View style={[styles.overlay, animatedOverlayStyles]} />
-          </TouchableWithoutFeedback>}
+        <TouchableWithoutFeedback onPress={this._onOverlayClick}>
+          <Animated.View
+            style={[drawerShown && styles.overlay, animatedOverlayStyles]}
```

Comment:
Good idea! The element should not have a height and width, but setting it either way would be better.